links.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "http://json-schema.org/draft/hyper-schema#",
+    "id": "http://json-schema.org/draft/links#",
+    "title": "Link Description Object",
+    "type": "object",
+    "required": [ "href" ],
```

Comment:
draft 4 version at https://github.com/json-schema-org/json-schema-org.github.io/blob/master/draft-04/links#L6 includes "rel" as required. Was this changed in draft 4? (Just checking)